plugins/tiddlywiki/jasmine/files/jasmine-core/lib/jasmine-core/jasmine.js

```diff
@@ -170,7 +170,7 @@ getJasmineRequireObj().base = function(j$, jasmineGlobal) {
    * Default number of milliseconds Jasmine will wait for an asynchronous spec to complete.
    * @name jasmine.DEFAULT_TIMEOUT_INTERVAL
    */
-  j$.DEFAULT_TIMEOUT_INTERVAL = 5000;
+  j$.DEFAULT_TIMEOUT_INTERVAL = 30000;
```

Comment:
Files within the `plugins/tiddlywiki/jasmine/files` directory are taken directly from the Jasmine release. Any changes here will be overwritten the next time Jasmine is upgraded. If the increased timeout is necessary we should explore whether it can be configured in the TiddlyWIki JS wrappers